lib/request_handler/base.rb

```diff
@@ -71,9 +71,9 @@ def parse_filter_params
       defaults = fetch_defaults('filter.defaults', {})
       defaults.merge(FilterParser.new(
         params:                params,
-        schema:                config.lookup!('filter.schema'),
-        additional_url_filter: config.lookup!('filter.additional_url_filter'),
-        schema_options:        execute_options(config.lookup!('filter.options'))
+        schema:                lookup!('filter.schema'),
+        additional_url_filter: lookup('filter.additional_url_filter'),
```

Comment:
I meant that defaults in the parser do not matter much if this call will always pass in `nil`. Have you checked this?